src/store/local-storage.ts

```diff
@@ -16,7 +17,7 @@ export const saveState = (state: RootState) => {
     try {
         const serializedState = JSON.stringify(state);
         localStorage.setItem("state", serializedState);
-        console.log(`State saved`);
+        console.log("State saved to local storage");
```

Comment:
Are  these console.logs intended to be kept?

Reply:
I think they are fine until we set up a production environment and get the configurations to disable them.
 
The other console logs were removed because we are now using a much improved system that always captures the dispatch -> next state and doesn't spam up the console or hinder performance. It's generally much easier to debug any kind of reducer than it was.

As we will likely be testing things like loading and saving states based on versions in the semi-near future they might as well stay in (we might even add some kind of state logger to replace this).